### PR TITLE
Add toggle to disable PostgreSQL secret generation

### PR DESCRIPTION
**Pull request by PenguinToast**
## Summary

This is useful when the PostgreSQL secret is managed outside of Helm.

Context: https://dagster.slack.com/archives/C014N0PK37E/p1616053327016800

## Test Plan

Tested by confirming `helmfile diff` shows no new Secret resource on our deployment

## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by alangenfeld** [comment]
sweet - thanks for the contribution!

File: helm/dagster/values.schema.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
         "postgresql": {
             "$ref": "#/definitions/PostgreSQL"
         },
+        "generatePostgresqlPasswordSecret": {
+            "title": "Generatepostgresqlpasswordsecret",
+            "type": "boolean"
+        },
```
Comment (alangenfeld) [inline helm/dagster/values.schema.json:18]: this json file is a generated artifact, 
the source of truth for this is https://github.com/dagster-io/dagster/blob/master/helm/dagster/schema/schema/charts/dagster/values.py

the cli for generating is at `helm/dagster/schema/schema/cli.py`

**Review by alangenfeld — CHANGES_REQUESTED** [review]
one test failure to fix - commented inline

File: helm/dagster/values.schema.json (hunk 1 above)
Comment (PenguinToast) [reply helm/dagster/values.schema.json:18]: Oh, cool -- fixed, thanks!

**Pull request merged**